scantools/scanners/navvis/iwconfig_parser.py

```diff
@@ -76,12 +81,14 @@ def parse_iwconfig(data):
         assert -127 <= float(wifi_sample['signal_level']) <= 127
         assert wifi_sample['frequency'] is not None
         assert wifi_sample['time_offset'] is not None
+        assert wifi_sample['ssid'] is not None
```

Comment:
did we verify we can do this? we've recently found that some WiFi samples on some other platforms do not contain ssid so flagging here whether we're certain this is always available

Reply:
For NavVis I have not seen any wifi sample without it. And I would argue we should also be strict here. Thanks for checking.